h2o-py/tests/testdir_misc/pyunit_export_file_multi.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import print_function
+from builtins import range
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.estimators.glm import H2OGeneralizedLinearEstimator
+import string
+import os
+import glob
+import random
+import pandas as pd
+from pandas.util.testing import assert_frame_equal
+
+'''
+Export file with h2o.export_file and compare with Python counterpart when re importing file to check for parity.
+This is a variation of a default h2o.export_file test. This tests makes sure that support for export to multiple
+'part' files is working. This test checks that when user specifies number of part files a directory is created
+instead of just a single file. It doesn't check the actual number of part files.
+'''
+
+def export_file_multipart():
+    pros_hex = h2o.upload_file(pyunit_utils.locate("smalldata/prostate/prostate.csv"))
+    pros_hex[1] = pros_hex[1].asfactor()
+    pros_hex[3] = pros_hex[3].asfactor()
+    pros_hex[4] = pros_hex[4].asfactor()
+    pros_hex[5] = pros_hex[5].asfactor()
+    pros_hex[8] = pros_hex[8].asfactor()
+
+    p_sid = pros_hex.runif()
+    pros_train = pros_hex[p_sid > .2, :]
+    pros_test = pros_hex[p_sid <= .2, :]
+
+    glm = H2OGeneralizedLinearEstimator(family="binomial")
+    myglm = glm.train(x=list(range(2, pros_hex.ncol)), y=1, training_frame=pros_train)
+    mypred = glm.predict(pros_test)
```

Comment:
Not sure why you need to spend time training a GLM model, if all you test is frame exporting. Can't you just export the original pros_hex? Or better yet, create a random frame using `h2o.create_frame` of desired dimension and export that? Ideally we want each test to be as small as possible...